src/util.c

```diff
@@ -125,15 +122,13 @@ int process_test_results(struct ntttcp_test_endpoint *tep)
 	tepr->cpu_ps_softirq_usage = (tepr->final_cpu_ps->softirq_time - tepr->init_cpu_ps->softirq_time) / cpu_ps_total_diff;
 
 	/* calculate for counters for xml log (compatible with Windows ntttcp.exe) */
-	tepr->throughput_Bps  = (double)total_bytes / test_duration;
-	tepr->total_bytes_MB  = (double)total_bytes / DECIMAL_BASED_UNIT_M;
+	tepr->throughput_Bps = (double)total_bytes / test_duration;
+	tepr->total_bytes_MB = (double)total_bytes / DECIMAL_BASED_UNIT_M;
 	tepr->throughput_MBps = tepr->total_bytes_MB / test_duration;
 	tepr->throughput_mbps = tepr->throughput_MBps * BYTE_TO_BITS;
-	tepr->cycles_per_byte = total_bytes == 0 ? 0 :
```

Comment:
I prefer to keep the old style for the last element of the operator.

Reply:
done